pandas/core/series.py

```diff
@@ -4579,14 +4577,9 @@ def _reduce(
                 if name in ["any", "all"]:
```

Comment:
Can we remove that? This raises in all cases

Reply:
the message is slightly different different

Reply:
Ah missed that, thanks